evm/verify-contract.js

```diff
@@ -101,24 +150,11 @@ async function processCommand(config, chain, options) {
         }
 
         case 'AxelarGateway': {
-            const gateway = contractFactory.attach(contractAddress);
-            const implementation = await gateway.implementation();
-            const auth = await gateway.authModule();
-            const tokenDeployer = await gateway.tokenDeployer();
-
-            const { addresses, weights, threshold } = await getEVMAddresses(config, chain.axelarId, {
-                keyID: chain.contracts.AxelarGateway.startingKeyIDs[0] || options.args || `evm-${chain.axelarId.toLowerCase()}-genesis`,
-            });
-            const authParams = [defaultAbiCoder.encode(['address[]', 'uint256[]', 'uint256'], [addresses, weights, threshold])];
-            const setupParams = defaultAbiCoder.encode(
-                ['address', 'address', 'bytes'],
-                [contractConfig.deployer, contractConfig.deployer, '0x'],
-            );
-
-            await verifyContract(env, chain.name, auth, [authParams], verifyOptions);
-            await verifyContract(env, chain.name, tokenDeployer, [], verifyOptions);
-            await verifyContract(env, chain.name, implementation, [auth, tokenDeployer], verifyOptions);
-            await verifyContract(env, chain.name, contractAddress, [implementation, setupParams], verifyOptions);
+            if (contractConfig.connectionType === 'amplifier') {
```

Comment:
If connectionType isn't present, throw an error that connectionType should be set to the expected values